### DISCUSSION TITLE
particle system bug?

### DISCUSSION THREAD

**Discussion by BaddRadish** [godotengine/godot-proposals#5263] [3D]
i think i found a bug in the particle system, version is 4a14.

please follow these steps IN ORDER to recreate:
1 GPUParticles3D
2 process material: new particles material
3 pass 1: new quad mesh
4 material: new standard material 3d
5 emission

can anyone verify this bug?

**Comment by Zireael07** [comment]
Bug reports go on Godot repo, not proposals repo
